src/utils/emailRainbow.ts

```diff
@@ -0,0 +1,43 @@
+import { debounce } from 'lodash';
+// @ts-expect-error
+import Mailer from 'react-native-mail';
```

Comment:
It's definitely not in scope of this PR, but this library doesn't seem to be maintained anymore.